Sources/HTMLEntities/String+HTMLEntities.swift

```diff
@@ -0,0 +1,884 @@
+    "&weierp;" : "\u{2118}",
+    "&image;" : "\u{2111}",
+    "&real;" : "\u{211C}",
+    "&trade;" : "\u{2122}",
+    "&alefsym;" : "\u{2135}",
+    "&larr;" : "\u{2190}",
+    "&uarr;" : "\u{2191}",
+    "&rarr;" : "\u{2192}",
+    "&darr;" : "\u{2193}",
+    "&harr;" : "\u{2194}",
+    "&crarr;" : "\u{21B5}",
+    "&lArr;" : "\u{21D0}",
+    "&uArr;" : "\u{21D1}",
+    "&rArr;" : "\u{21D2}",
+    "&dArr;" : "\u{21D3}",
+    "&hArr;" : "\u{21D4}",
+    "&forall;" : "\u{2200}",
+    "&part;" : "\u{2202}",
+    "&exist;" : "\u{2203}",
+    "&empty;" : "\u{2205}",
+    "&nabla;" : "\u{2207}",
+    "&isin;" : "\u{2208}",
+    "&notin;" : "\u{2209}",
+    "&ni;" : "\u{220B}",
+    "&prod;" : "\u{220F}",
+    "&sum;" : "\u{2211}",
+    "&minus;" : "\u{2212}",
+    "&lowast;" : "\u{2217}",
+    "&radic;" : "\u{221A}",
+    "&prop;" : "\u{221D}",
+    "&infin;" : "\u{221E}",
+    "&ang;" : "\u{2220}",
+    "&and;" : "\u{2227}",
+    "&or;" : "\u{2228}",
+    "&cap;" : "\u{2229}",
+    "&cup;" : "\u{222A}",
+    "&int;" : "\u{222B}",
+    "&there4;" : "\u{2234}",
+    "&sim;" : "\u{223C}",
+    "&cong;" : "\u{2245}",
+    "&asymp;" : "\u{2248}",
+    "&ne;" : "\u{2260}",
+    "&equiv;" : "\u{2261}",
+    "&le;" : "\u{2264}",
+    "&ge;" : "\u{2265}",
+    "&sub;" : "\u{2282}",
+    "&sup;" : "\u{2283}",
+    "&nsub;" : "\u{2284}",
+    "&sube;" : "\u{2286}",
+    "&supe;" : "\u{2287}",
+    "&oplus;" : "\u{2295}",
+    "&otimes;" : "\u{2297}",
+    "&perp;" : "\u{22A5}",
+    "&sdot;" : "\u{22C5}",
+    "&lceil;" : "\u{2308}",
+    "&rceil;" : "\u{2309}",
+    "&lfloor;" : "\u{230A}",
+    "&rfloor;" : "\u{230B}",
+    "&lang;" : "\u{2329}",
+    "&rang;" : "\u{232A}",
+    "&loz;" : "\u{25CA}",
+    "&spades;" : "\u{2660}",
+    "&clubs;" : "\u{2663}",
+    "&hearts;" : "\u{2665}",
+    "&diams;" : "\u{2666}",
+
+    // Markup-significant and internationalization characters
+    // https://www.w3.org/TR/html4/sgml/entities.html#h-24.4.1
+    "&quot;" : "\u{0022}",
+    "&amp;" : "\u{0026}",
+    "&lt;" : "\u{003C}",
+    "&gt;" : "\u{003E}",
+    "&OElig;" : "\u{0152}",
+    "&oelig;" : "\u{0153}",
+    "&Scaron;" : "\u{0160}",
+    "&scaron;" : "\u{0161}",
+    "&Yuml;" : "\u{0178}",
+    "&circ;" : "\u{02C6}",
+    "&tilde;" : "\u{02DC}",
+    "&ensp;" : "\u{2002}",
+    "&emsp;" : "\u{2003}",
+    "&thinsp;" : "\u{2009}",
+    "&zwnj;" : "\u{200C}",
+    "&zwj;" : "\u{200D}",
+    "&lrm;" : "\u{200E}",
+    "&rlm;" : "\u{200F}",
+    "&ndash;" : "\u{2013}",
+    "&mdash;" : "\u{2014}",
+    "&lsquo;" : "\u{2018}",
+    "&rsquo;" : "\u{2019}",
+    "&sbquo;" : "\u{201A}",
+    "&ldquo;" : "\u{201C}",
+    "&rdquo;" : "\u{201D}",
+    "&bdquo;" : "\u{201E}",
+    "&dagger;" : "\u{2020}",
+    "&Dagger;" : "\u{2021}",
+    "&permil;" : "\u{2030}",
+    "&lsaquo;" : "\u{2039}",
+    "&rsaquo;" : "\u{203A}",
+    "&euro;" : "\u{20AC}"
+]
+
+private let amp: Character = "&"
+private let semicolon: Character = ";"
+
+public extension String {
```

Comment:
I think you should add some overall API description to what you are trying to do in this extension.